### PR TITLE
Add message for users with no visible sets.

### PR DESCRIPTION
**Pull request by somiaj**
Show a message for users viewing the "Assignments" page when they have no visible sets. This also hides the show by buttons when no sets are visible.

This addresses #2550.

### CODE REVIEW

File: templates/ContentGenerator/ProblemSets.html.ep [hunk 1]
```diff
@@ -21,35 +21,41 @@
...
-		   	>
-			<%= maketext('Show By Date') =%>
-		</button>
-		<button id="show-by-type-btn" type="button" class="btn btn-outline-primary"
-			data-default-title="<%= maketext('Regular Assignments') %>"
-			data-test-title="<%= maketext('Tests/Quizzes') %>"
-			>
-			<%= maketext('Show By Type') =%>
-		</button>
+% #
```
Comment (drgrice1) [inline templates/ContentGenerator/ProblemSets.html.ep:24]: There is no comment here so the `#` is not needed.

**Review by drgrice1 — APPROVED** [review]
Looks good.

File: templates/ContentGenerator/ProblemSets.html.ep (hunk 1 above)
Comment (somiaj) [reply templates/ContentGenerator/ProblemSets.html.ep]: Thanks, I was going to put a comment about only showing the sets when they exist, but changed my mind, though the if statement was clear enough.

**Review by Alex-Jordan — APPROVED** [review]

**Comment by Alex-Jordan** [comment]
Perhaps this should have a hotfix variant. It is addressing a shortcoming of a feature that was introduced with 2.19, not really adding a new feature.

**Comment by drgrice1** [comment]
A hot fix would be fine for this.

**Pull request merged**